src/orders/components/OrderFulfillPage/OrderFulfillPage.tsx

```diff
@@ -162,7 +162,10 @@ const OrderFulfillPage: React.FC<OrderFulfillPageProps> = props => {
     OrderFulfillStockInput[]
   >(
     order?.lines
-      .filter(line => getRemainingQuantity(line) > 0)
+      .filter(
+        line => getRemainingQuantity(line) > 0
+        // && line.variant !== null
```

Comment:
I think this should be uncommented

Reply:
good catch. uncommented this one and added another in line 316